Source/Cogworks.Umbraco.Essentials/Builders/Interfaces/IResponsiveImageBuilder.cs

```diff
@@ -19,7 +19,9 @@ public interface IResponsiveImageBuilder
             int? height = null,
             bool includeRetina = true,
             bool enableWebP = false,
-            int? quality = null);
+            int? quality = null,
+            bool isLazy = true,
+            bool addDefaultDimensions = false);
```

Comment:
I would rename it to `defaultDimensions` or `enableDefaultDimensions` 🤔 What do you think ?

Reply:
Agree -> addDefaultDimension sounds like a function.